### PR TITLE
initial template structure

### PR DESCRIPTION
**Pull request by VikSafiullina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,48 @@
-webrpc-gen Python templates
+webrpc-gen Typescript templates
```
Comment (VojtechVitek) [inline README.md]: Python?

**Review by VojtechVitek — APPROVED** [review]
So far so good @holowcost!

Does the generated code work? Can I try it out?

File: README.md (hunk 1 above)
Comment (VikSafiullina) [reply README.md]: ooops, will fix

**Review by VojtechVitek — APPROVED** [review]
LGTM

**Pull request merged**